modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -148,7 +147,9 @@ public Connection createConnection(String queryString) throws SQLException {
         final Driver jdbcDriverInstance = getJdbcDriverInstance();
 
         try {
-            return Unreliables.retryUntilSuccess(120, TimeUnit.SECONDS, () -> jdbcDriverInstance.connect(url, info));
+            return Unreliables.retryUntilSuccess(120, TimeUnit.SECONDS, () ->
```

Comment:
Should this 120 also go into a `protected` method?

Reply:
Good call - done.